### PR TITLE
Store Item Creation Docs

### PR DESCRIPTION
**Pull request by joshholat**

### CODE REVIEW

File: api_v1.md [hunk 1]
```diff
@@ -1280,6 +1283,130 @@ offset
...
+
+    a parent object containing all information to be associated with this store item
+
+`store_item[options]` _(required for store_item[type] set to PHYSICAL or EXPERIENCE)_
+
+    an array of objects representing each option (for a T-shirt each option may represent a size, a color, or a combination of the two, etc)
+
+    each element in this array may contain the following fields:
+
+    `name` _(required)_
```
Comment (joshholat) [inline api_v1.md]: @jdhartley Is there any way we can update the developers section of our site to style indented things like this differently?

Comment (acdameli) [inline api_v1.md]: the only alternative I could think of was something like:

    `store_item[options][][name]` _(required)_

Comment (acdameli) [inline api_v1.md]: but that's super awkward when that's an array of objects as in the case of options.

File: api_v1.md [hunk 2]
```diff
@@ -1280,6 +1283,130 @@ offset
...
+
+    `weight` _(required for store_item[type] set to PHYSICAL)_
+
+        a non-negative decimal number indicating the weight in ounces of this option (used for shipping price calculation)
+
+    `height` _(required for store_item[type] set to PHYSICAL)_
+
+        a non-negative decimal number indicating the height in inches of this option (used for shipping price calculation)
+
+    `width` _(required for store_item[type] set to PHYSICAL)_
```
Comment (acdameli) [inline api_v1.md]: turns out this is not, strictly speaking, required. Defaults to 0, same with height and length.

File: api_v1.md [hunk 3]
```diff
@@ -1365,6 +1368,170 @@ offset
...
+store\_item[fulfiller\_id] _(required for type set to PHYSICAL)_
+
+    an integer ID for a fulfiller already configured in StageBloc which is associated with this account
+
+### Example Request
+
+	{
+	    "store_item": {
+	        "price": 10.5,
+	        "sort_order": ,
```
Comment (stagebloc-iceman) [inline api_v1.md]: @joshholat This seems to be missing a value and isn't listed in the documentation above.

File: api_v1.md [hunk 4]
```diff
@@ -1365,6 +1368,170 @@ offset
...
+	            {
+	                "name": "minimal option",
+	                "weight": 4.5
+	            }
+	        ],
+	        "shipping_price_handlers": [
+	            {
+	                "name":"USPS",
+	                "price":4.5,
+	                "media-mail":true
```
Comment (stagebloc-iceman) [inline api_v1.md]: @joshholat @acdameli Is it too late to ninja-edit this to use an underscore like all other properties?

File: api_v1.md (hunk 3 above)
Comment (joshholat) [inline api_v1.md]: @stagebloc-iceman I feel like setting this via the API is just going to make things more confusing for now, isn't really necessary to do in the API.

**Pull request merged**